### PR TITLE
Feat/implement tick and call server updates zappy gui

### PR DESCRIPTION
**Pull request by mathieurobert1**
I handle the tick of the server in the GameData.
Each tick, the gui ask the following things:
- content of the map
- the tick
- the map size
- the teams names

### CODE REVIEW

**Review by Thomaltarix — DISMISSED** [review]
Clean

File: gui/include/Network/Network.hpp [hunk 1]
```diff
@@ -85,6 +85,13 @@ class Gui::Network {
          */
         const std::string listenServer();
 
+        /**
+         * @brief Send a message to the Server.
+         *
+         * @param message to send to the server
```
Comment (Njord201) [inline gui/include/Network/Network.hpp:91]: Missing one point

Comment (Njord201) [reply gui/include/Network/Network.hpp:91]: And the capital letter

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by Njord201 — APPROVED** [review]

**Review by Thomaltarix — APPROVED** [review]
Clean

**Pull request merged**